package.json

```diff
@@ -221,7 +221,8 @@
     "rm": "node bin/rm.js",
     "preshrinkwrap": "npm prune",
     "postshrinkwrap": "node bin/trim-shrinkwrap.js && touch node_modules",
-    "prestart": "npm run -s install-if-deps-outdated && node bin/welcome.js && ( check-node-version --package || exit 0 )",
+    "check-versions": "check-node-version --package || ( echo '\n!!! Calypso will only work with the versions listed. Please set up your environment accordingly.\n' && exit 1 )",
```

Comment:
*If* we decide to merge this, `"check-node-version --package"` will just fail with an error, no need to do all that `|| echo && exit 1` contraption.

Reply:
Yes but part of the initiative for this PR was to make it easier for newbies and provide a decent error message.